### PR TITLE
Add hints how to make your app look like the admin

### PR DESCRIPTION
**Pull request by JoshuaBehrens**
I published an app and wanted it to be nice. It was like the Shopware 5 lightweight modules. No style or tooling given. Since the pre-compiled packages are available it is now super easy to match the styles :) I tell you how I was able to pull it off

### CODE REVIEW

File: guides/plugins/apps/administration/add-custom-modules.md [hunk 1]
```diff
@@ -141,3 +141,10 @@ To avoid mixing other modules with your main module, we decided to separate the
 
 This feature is not compatible with themes as they will always open the theme config by default.
 
+## Admin design compatibility
```
Comment (keulinho) [inline guides/plugins/apps/administration/add-custom-modules.md:144]: We are aware of the difficulties of getting the look&feel of your iframes to match the rest of the administratrion, we have a solution in mind, but are not sure when we can deliver on that. For the meantime i am fine with this creative approach. @Christian-Rades what do you think?

Comment (Christian-Rades) [reply guides/plugins/apps/administration/add-custom-modules.md:144]: Agreed, I think it's a good Idea :+1:

Comment (JoshuaBehrens) [reply guides/plugins/apps/administration/add-custom-modules.md:144]: "Creative approach" :) I don't see where programming isn't creative

**Review by keulinho — APPROVED** [review]

**Comment by keulinho** [comment]
Thanks for your contribution @JoshuaBehrens :tada: :blue_heart:

**Pull request merged**